code.js

```diff
@@ -0,0 +1,69 @@
+const panels = document.querySelectorAll('.panel');
+const playButton = document.querySelector('.play i');
+const stopButton = document.querySelector('.stop i');
+const replayButton = document.querySelector('.replay i');
+const song = document.querySelector('.chinasong');
+const meters = document.querySelectorAll('svg[data-value] .meter');
+
+playButton.addEventListener('click', playMusic);
+stopButton.addEventListener('click', stopMusic);
+replayButton.addEventListener('click', rePlayMusic);
+for (let i = 0; i < panels.length; i++) {
+    panels[i].addEventListener('click', setActive);
+}
+setInterval(renderProgressBar, 1000);
+
+function playMusic(event) {
+    if (song.currentTime === 0) {
+        song.play();
+        event.target.classList.remove('fa-play');
+        event.target.classList.add('fa-pause');
+    } else if (song.paused) {
```

Comment:
I think it's not necessary, you can combine both block in your condition
if (song.currentTime === 0 || song.paused) {
        song.play();
        event.target.classList.remove('fa-play');
        event.target.classList.add('fa-pause');
    }

Reply:
done